### PR TITLE
Flake: Attempt at fixing Personalize configuration

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**

**What does this PR do / why we need it:**

**Which issue(s) this PR fixes:**

Fixes #

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Comment by valaparthvi** [comment]
Podman failure:
```sh
  ✓  Deleting resources from podman [12s]
  [odo] The component "pfydfi" is successfully deleted from podman
  [odo] I0522 04:09:15.094697  139220 podman.go:195] Deleted volume odo-shared-data-pfydfi-app
  Setting current dir to: /home/runner/work/odo/odo/tests/integration
  Deleting dir: /tmp/975753024
  Deleting dir: /tmp/3047114366
  << Timeline

  [FAILED] Expected
      <string>: Searching resources to delete, please wait...
      No resource found for component "pfydfi"
      
      
  to contain substring
      <string>: - pfydfi
  In [It] at: /home/runner/work/odo/odo/tests/integration/cmd_delete_test.go:279 @ 05/22/23 04:08:58.189
------------------------------
SSSSSSSSSS•SSS••SSS•SSSSSS•SSSSSSSSS••SS•SSSS•SSSSSSSSSSS•S••SSSSSSSSSSSSS•SSSSS•SSSS•SSSSSS•SSSS•S•SSSS••SS•SSSS•SSS•S•SSSS•SSSSSSSSSSS•SSSSSSSSSSSSS•SSSSS•SSSSSSSSSSSSSSSSSSS••SSSSS•SSSSSSS•SSS••SSSSSSSSSSSSSSS•SSS•SSSSSSS•SSS•SS•SSS•SSSSSS•S•••SSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSS•••S•SSSSSSSSSSSSSS•SSSSSS•SSSSSSSSSSS•SSSSSSSS••SSSSS•SSSS••SSSS•S•S•SSSSSS•SS•SSSSS•SSSSSSSSSSSS•SSSSSSSSSSS•SSSSSSSSSSSS•SSS•SS•S•SSSSSS•S•SSS•SSSSSSSSSSSSSS••SSSS•SSSS•S•SSSSSS••••••••••

Summarizing 1 Failure:
  [FAIL] odo delete command tests when a component is bootstrapped when the component is deployed in DEV mode and dev mode stopped when the component is deleted using its name (and namespace) from another directory (running-in="") [It] should have deleted the component [podman]
```
Rerunning the job.

**Comment by feloy** [comment]
The test is still flaky:

```
 > Delete environment variable "DEBUG_PORT"
  ? What configuration do you want change? Delete environment variable "DEBUG_PORT"  [Use arrows to move, type to filter]
  > Delete environment variable "DEBUG_PORT"
  ? What configuration do you want change? Delete environment variable "DEBUG_PORT"
  ? What configuration do you want change?  [Use arrows to move, type to filter]
  > NOTHING - configuration is correct
    Delete port "5858"
    Delete port "5000"
    Add new port
    Delete environment variable "DEBUG_PROJECT_PORT"
    Add new environment variable
  
  ======================
  Unexpected error:
      <*xpty.errPassthroughTimeout | 0xc001d6c010>: {
          error: <*errors.errorString | 0xc001d6c000>{
              s: "passthrough i/o timeout",
          },
      }
      passthrough i/o timeout
  occurred
  In [It] at: /go/odo_1/tests/helper/helper_interactive.go:111 @ 06/06/23 08:31:53.33
```

File: tests/helper/helper_interactive.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func PressKey(ctx InteractiveContext, c byte) {
 }
 
 func ExpectString(ctx InteractiveContext, line string) {
-	res, err := ctx.cp.Expect(line, 120*time.Second)
+	res, err := ctx.cp.Expect(line, 180*time.Second)
```
Comment (valaparthvi) [inline tests/helper/helper_interactive.go:109]:
```suggestion
	res, err := ctx.cp.Expect(line, 120*time.Second)
```

**Review by feloy — APPROVED** [review]
Thanks for this fix @valaparthvi

**Comment by feloy** [comment]
/override windows-integration-test/Windows-test

Not related

**Pull request merged**